docs/workforce-management-framework/topics/con-PassportAuth-Module.adoc

```diff
@@ -0,0 +1,38 @@
+[]
+= {WFM-RC-NameShort} Passport Auth
+
+{WFM-RC-NameShort} authentication and authorization module using link:http://passportjs.org[PassportJS].
```

Comment:
What I mean here is to do less copy paste and try to fiugre out some good format for this documentation.
Explain why module is being used.

I think having less code samples and more explanations and references to documentation (not existing, but we will generate that) will be more effective here. By the end of the day we will point people to example app.